packages/malloy-vscode/package.json

```diff
@@ -342,6 +346,8 @@
     "@types/jsdom": "^16.2.11",
     "@types/mocha": "^8.2.2",
     "@types/prismjs": "^1.26.0",
+    "@types/node": "^12.19.8",
```

Comment:
I think this probably shouldn't be added at this point, since it will conflict with the node 16 binding.